src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,187 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static User user;
+    private static RegistrationServiceImpl registrationService;
+
+    private static final int DEFAULT_VALID_AGE = 20;
+    private static final String DEFAULT_VALID_LOGIN = "some_login";
+    private static final String DEFAULT_VALID_PASSWORD = "12345678";
+    private static final int INVALID_AGE = 15;
+    private static final String INVALID_LOGIN = "log";
+    private static final String INVALID_PASSWORD = "123";

```

Comment:
In your test classes that is OK if you use literals directly instead of constants. But, if you already have created a lot of constants, you can move them to a separate class, e.g. RegistrationServiceTestUtils or RegistrationServiceTestConstants